src/raspgierry_pi.py

```diff
@@ -1,36 +1,41 @@
-import pygame, sys
-from pygame.locals import *
-from controls.joystick import *
+from src.gui.menu import *
+from src.gui.hud import *
+
 
 class RaspgierryPi:
     def __init__(self):
         pygame.init()
-        self._display = {'width': 320, 'height': 480}
-        native_screen = pygame.display.Info()
-        if native_screen.current_w < self._display['width'] or native_screen.current_h < self._display['height']:
-            # TODO sk: Add error screen for minimum display when available
-            sys.exit(1)
-        self._display_ratio = native_screen.current_w /  self._display['width']
-        self._window = pygame.display.set_mode((0,0), FULLSCREEN)
+        self._window = pygame.display.set_mode((LayoutRsc.WINDOW_WIDTH,  LayoutRsc.WINDOW_HEIGHT), RESIZABLE)
         pygame.display.set_caption('Raspgierry Pi')
+        try:
+            self._joystick = Joystick(0)
+        except pygame.error:
+            print("Couldn't detect valid joystick")
+            sys.exit(1)
+        self._menu = Menu()
+        self._time_per_frame = 0.1
```

Comment:
```suggestion
        self._frames_per_second = 0.1 //recalculate
```

Reply:
It's a common design pattern for video games called "Fixed Time Step".  It always uses time per frame, not frames per second because of its logic